sdk/lib/opentelemetry/sdk/trace/export/multi_span_exporter.rb

```diff
@@ -29,8 +29,7 @@ def export(spans)
               begin
                 merge_result_code(result_code, span_exporter.export(spans))
               rescue => e # rubocop:disable Style/RescueStandardError
-                # TODO: log instead of puts
-                puts "WARNING: exception thrown by export - #{e}"
+                logger.warn("exception raised by export - #{e}")
```

Comment:
@fbogsany is this meant to be `OpenTelemetry.logger`? 

I've just rebased #98 against master with this PR merged, and this is now failing with "undefined local variable or method `logger'" in my newly added tests.

Reply:
Yes, it should be. Is there a reason why `OpenTelemetry.logger` wouldn’t be accessible as `logger` here? CI passed for this branch pre-merge.